tools/reviewer/job/sync/GithubReader.java

```diff
@@ -195,53 +195,57 @@ private Thread getCodeThreadByFilenameAndPosition(
                 .filter(comment -> comment.getPosition() == position)
                 .collect(Collectors.toList()));
 
-    Thread.Builder thread = Thread.newBuilder();
+    GithubComment firstComment;
```

Comment:
Please add a comment for why you use the first comment - that you are trying to fit several comments into a thread, and you assume all comments have the same values for these fields (is that really true?)

Reply:
Done.